girder_worker/plugins/docker/docker-gc

```diff
@@ -0,0 +1,267 @@
+#!/bin/bash
+
+# Taken from https://github.com/spotify/docker-gc
```

Comment:
Could you be more explicit by for example specifying the exact revision. 

E.g:

Taken from spotify/docker-gc@fa99444